OPHD/UI/StringTable.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <NAS2D/Renderer/Renderer.h>
+#include <NAS2D/Renderer/Color.h>
+#include <NAS2D/Renderer/Point.h>
+#include <NAS2D/Renderer/Vector.h>
+#include <string>
+#include <vector>
+#include <cstddef>
+
+// Draw a 2 dimensional table of text. Determine cell size based on inserted text, font, and padding. Only allows one line of text per cell.
+class StringTable
+{
+public:
+    using CellCoordinate = NAS2D::Point<std::size_t>;
+
+    enum class Justification
+    {
+        Left,
+        Right
+    };
+
+    // Set default fonts in constructor
+    StringTable(std::size_t columns, std::size_t rows);
+
+    void draw(NAS2D::Renderer& renderer) const;
+
+    void setPosition(NAS2D::Vector<float> position);
```

Comment:
I noticed in other NAS2D classes, Set/Get Position is completed as a function overload:

```
void position(NAS2D::Vector<float> position)
NAS2D::Vector<float> position()
```

Is the overload preferred to get/set here?

Reply:
It would be stylistically closer to existing code.

Though with that said, the refactoring has been making some of those overloads disappear. I'm not sure it's a style we necessarily want to maintain, though I'm also a bit hesitant to deviate from it.

Of course, if `StringTable` were to derive from `Control`, it would obtain such methods through inheritance.

Reply:
OK, I'll leave it up to you or @ldicker83 if you would prefer StringTable to inherit from 'Control'. I wasn't sure it needed full support for things like visibility, focus, textchange events, etc. 

Otherwise, I'll try to match the existing terminology. It is easy to change the function names at a later date.

-Brett

Reply:
This reminds me, I would like to pare down the functionality for `Control`. It contains a few things that aren't truly generic, such as text change. Not all controls have text or focus.

----

I've noticed a lot of existing code prefixes member variables names with "m", which is one way to avoid name collisions. In my own projects I usually try to avoid name prefixes, though it's pretty widespread in this project.